autosklearn/automl.py

```diff
@@ -1834,15 +1834,52 @@ def get_models_with_weights(self):
         return self.ensemble_.get_models_with_weights(self.models_)
 
     def show_models(self):
```

Comment:
Can we add a return type here
```python
def show_models(self) -> Dict[str, Any]:
    ...
```

It's not the best type hint we can give but it least lets a used who has any nice IDE know that what they receive is a dictionary from 'str' to something.